launch_ros/launch_ros/actions/node.py

```diff
@@ -180,6 +181,9 @@ def __init__(
         wildcard namespace (`/**`) and other specific parameter declarations
         may overwrite it.
 
+        Using `ros_arguments` is equivalent to using `arguments` with a
+        prepended '--ros-args' item.
```

Comment:
@christophebedard I see the convenience BUT is it necessary? Most ROS arguments can be given in an idiomatic way  (e.g. via `remappings`) already. If we need `--log-level`, I'd argue we should add it. CC @wjwwood @ivanpauno.

Reply:
I found #60 while answering an answers.ros.org question: https://answers.ros.org/question/380883/specify-log-level-on-ros2-node-in-launch-file/#380965. In this case, it seemed to confuse the person. They weren't sure whether they needed to use `--` beause `launch_ros` appends `--ros-args ...` too (even if in practice `rcl` handles it just fine). I think having `ros_arguments` would make it much simpler, but yeah it's up to you the maintainers :stuck_out_tongue_closed_eyes:

> If we need `--log-level`

you mean like a `LogLevel` substitution?

Reply:
> I found #60 while answering an answers.ros.org question: https://answers.ros.org/question/380883/specify-log-level-on-ros2-node-in-launch-file/#380965. In this case, it seemed to confuse the person.

I see. I was wondering about the use case. If not having to write out `--ros-args` helps people, I'm onboard.

> you mean like a LogLevel substitution?

Like a `log_level` argument that can be a substitution. But that's a separate PR.

Reply:
Seems like we can handle things better if the user explicitly tells us when something is a ros argument, so +1 from me.